### PR TITLE
[NetBeans 2110] Update netbeans.conf for 11.0 release

### PR DESCRIPTION
**Pull request by geertjanw**

### CODE REVIEW

File: nb/ide.launcher/netbeans.conf [hunk 1]
```diff
@@ -62,7 +62,7 @@ netbeans_default_cachedir="${DEFAULT_CACHEDIR_ROOT}/dev"
...
...es -J--add-opens=java.base/java.net=ALL-UNNAMED -J--add-opens=java.base/java.lang.ref=ALL-UNNAMED -J--add-opens=java.base/java.lang=ALL-UNNAMED -J--add-opens=java.base/java.security=ALL-UNNAMED -J--add-opens=java.base/java.util=ALL-UNNAMED -J--add-opens=java.desktop/javax.swing.plaf.basic=ALL-UNNAMED -J--add-opens=java.desktop/javax.swing.text=ALL-UNNAMED -J--add-opens=java.desktop/javax.swing=ALL-UNNAMED -J--add-opens=java.desktop/java.awt=ALL-UNNAMED -J--add-opens=java.desktop/java.awt.event=ALL-UNNAMED -J--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED -J--add-opens=jdk.jshell/jdk.jshell=ALL-UNNAMED -J--add-modules=jdk.jshell -J--add-exports=java.desktop/sun.awt=ALL-UNNAMED -J--add-exports=java.desktop/java.awt.peer=ALL-UNNAMED -J--add-exports=java.desktop/com.sun.beans.editors=ALL-UNNAMED -J--add-exports=java.desktop/sun.swing=ALL-UNNAMED -J--add-exports=java.desktop/sun.awt.im=ALL-UNNAMED -J--add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED -J--add-exports=java.management/sun.management=ALL-UNNAMED -J--add-exports=java.base/sun.reflect.annotation=ALL-UNNAMED -J--add-exports=jdk.javadoc/com.sun.tools.javadoc.main=ALL-UNNAMED -J-XX:+IgnoreUnrecognizedVMOptions"
```
Comment (lkishalmi) [inline nb/ide.launcher/netbeans.conf]: What about the netbeans.logger.console, AFAIK it was removed for releases from 10.0?

Comment (geertjanw) [reply nb/ide.launcher/netbeans.conf]: I don't know about that..

Comment (lkishalmi) [reply nb/ide.launcher/netbeans.conf]: Well it seems you are growing old man!
https://issues.apache.org/jira/browse/NETBEANS-1313
:wink:

Comment (geertjanw) [reply nb/ide.launcher/netbeans.conf]: Ah, yes, that one. :-) But do we agree with it? I like info in the console.

Comment (lkishalmi) [reply nb/ide.launcher/netbeans.conf]: Well, I like it there as well. I do not know where that issue was coming from originally. Maybe from this one: https://issues.apache.org/jira/browse/NETBEANS-1308 but that is a different one.
I'd keep the console logs though.

Comment (geertjanw) [reply nb/ide.launcher/netbeans.conf]: OK. And the earlier PR failed I believe because this one ran into that one.

**Pull request merged**